integration/simulation/transaction_injector.go

```diff
@@ -59,14 +59,15 @@ func NewTransactionInjector(
 	for i := 0; i < numberWallets; i++ {
 		wallets[i] = wallet_mock.New()
 	}
+	interrupt := int32(0)
 
 	return &TransactionInjector{
 		wallets:          wallets,
-		avgBlockDuration: avgBlockDuration,
+		avgBlockDuration: avgBlockDuration * time.Microsecond, // nolint:durationcheck
```

Comment:
doesn't this multiply a duration with another duration?

Reply:
I know right - I had to go and double-check it too https://github.com/obscuronet/obscuro-playground/blob/eeed5c2c3497dae9e5d5e7caa538abb8580306d4/integration/simulation/transaction_injector.go#L173

But if we're using the actual block production time it bombs out. 
I didn't want to tinker with the values (40 Microseconds/250 Microseconds) bc I don't have a grasp of what they should be. So I just replicated the existing code.

Should I bump the the 40/250 Microseconds to Milliseconds ?

Reply:
Shouldn't it just be `* 1000`? That's the same but clearer.

Reply:
Good point 👍